.github/workflows/build.yml

```diff
@@ -64,4 +64,4 @@ jobs:
         if: github.event_name == 'push'
         run: |
           echo ${{ secrets.DOCKER_PASSWORD }} | docker login -u ${{ secrets.DOCKER_USERNAME }} --password-stdin
-          REPO=kubespheredev TAG=${GITHUB_REF#refs/heads/} make container-push
+          REPO=kubespheredev TAG=${GITHUB_REF#refs/*/} make container-push
```

Comment:
please modify https://github.com/kubesphere/kubesphere/blob/master/.github/workflows/build-multiarch.yaml too and cherry-pick this pr

Reply:
BTW, what's the value of `TAG ` if we run the workflow in tag `v3.1.1-rc.1`?

Reply:
```bash
GITHUB_REF='refs/tags/v3.1.1-rc.1'
echo ${GITHUB_REF#refs/*/}
```

Reply:
> please modify https://github.com/kubesphere/kubesphere/blob/master/.github/workflows/build-multiarch.yaml too and cherry-pick this pr

done